src/core/navigation/navigation.view.js

```diff
@@ -15,15 +15,23 @@ export class NavigationView extends View {
 		this.shortcutOff = shortcut.register('navigation', navigation.commands);
 
 		this.blur = new Command({
-			execute: (row, column) => table.body.cell(row, column).removeClass(`${GRID_PREFIX}-focus`),
+			execute: (row, column) =>{
+				table.body.cell(row, column).removeClass(`${GRID_PREFIX}-focus`);
```

Comment:
this should be done outside the grid in excel example, if it's not possible we will add this future,
for now try to use style API